package.json

```diff
@@ -2,8 +2,12 @@
   "name": "root",
   "private": true,
   "devDependencies": {
+    "@parcel/packager-ts": "2.9.3",
```

Comment:
Do parcel packages need to be installed at the root and at the package level? My knowledge of how lerna repos are organized is fairly limited...

Reply:
They shouldn't be necessary, but I've had a bear of a time keeping it from happen. One of parcel's selling points is that it automatically installs the plugins it needs to load the code it finds (based on the entrypoint we specify). But for some reason it installs packages in the root, not in the leaf. I'm not entirely sure why TBH, but I'll see if I can get rid of these here. AT the very least, we don't need these in both `./package.json` and `./packages/ui-components/package.json`.

Reply:
Goodness, so I managed to fix this but it required me to add a text file at `packages/ui-components/.hg` (yes, seriously) and I'm not sure if that's worse 😓

Reply:
I don't think it's worse! The comments in the file are very clear 🙏